src/app_errors/mod.rs

```diff
@@ -145,9 +148,47 @@ pub enum AppErrorKind {
     },
 
     /// An error for when an error happens on a request to the db.
-    #[fail(display = "{:?}", _0)]
+    #[fail(display = "{}", _0)]
     DbError(String),
 
+    /// An error for when an error happens on the queues process.
+    #[fail(display = "{}", _0)]
+    QueueError(String),
+    /// An error for when we get an invalid notification type in the queues
+    /// process.
+    #[fail(display = "Invalid notification type.")]
```

Comment:
Tiny nit: I'm not a fan of including a terminating period in our error strings fwiw. None of our other repos do it and it may be incorrect in the context of the receiving code, if it wants to display the message as part of a broader error string.